src/tool/hpcrun/gpu/gpu-correlation-id.h

```diff
@@ -52,7 +52,11 @@
 
 #include <stdint.h>
 
+#define GPU_CORRELATION_ID_START 0x8000000000000001
```

Comment:
There are other correlation ids, such as OMPT correlation ids, which start from 0. Since 0x8000000000000001 has been used from a long time ago, I'd rather keep the same value.